### PR TITLE
chore: Add proper packages description

### PR DESCRIPTION
**Pull request by Mielek**

### CODE REVIEW

File: src/Compiler/Compiler.csproj [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
+
+    <PropertyGroup>
+        <PackageId>Azure.ApiManagement.PolicyToolkit.Compiler</PackageId>
+        <PackageVersion>0.0.1</PackageVersion>
+        <Authors>Microsoft</Authors>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+        <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+        <RepositoryUrl>https://github.com/Azure/azure-api-management-policy-toolkit</RepositoryUrl>
+        <Description>Azure API Management Policy Toolkit compiler is a dotnet tool which is transforming policy document created in code to XML.</Description>
```
Comment (tomkerkhove) [inline src/Compiler/Compiler.csproj:11]:
```suggestion
        <Description>Azure API Management Policy Toolkit compiler is a dotnet tool allowing you to transform policy document(s) from C# code to XML.</Description>
```

File: src/Compiler/Compiler.csproj [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
+        <PackageTags>azure api-gateway api-management policy policies policy-toolkit</PackageTags>
```
Comment (tomkerkhove) [inline src/Compiler/Compiler.csproj:12]: Should you not use `;`?

File: src/Pack/Policy/Policy.csproj [hunk 3]
```diff
@@ -1,60 +1,56 @@
...
-<!--    <PackageIconUrl>http://ICON_URL_HERE_OR_DELETE_THIS_LINE</PackageIconUrl>-->
-<!--    <RepositoryUrl>http://REPOSITORY_URL_HERE_OR_DELETE_THIS_LINE</RepositoryUrl>-->
-<!--    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>-->
-<!--    <Description>Azure.ApiManagement.PolicyToolkit</Description>-->
-<!--    <PackageReleaseNotes>Summary of changes made in this release of the package.</PackageReleaseNotes>-->
-<!--    <Copyright>Copyright</Copyright>-->
-<!--    <PackageTags>Azure.ApiManagement.PolicyToolkit</PackageTags>-->
-    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <PropertyGroup>
+        <PackageId>Azure.ApiManagement.PolicyToolkit</PackageId>
```
Comment (tomkerkhove) [inline src/Pack/Policy/Policy.csproj:4]: I'd add something at the end, given testing is doing something similar
```suggestion
        <PackageId>Azure.ApiManagement.PolicyToolkit.Policies</PackageId>
```

File: src/Pack/Policy/Policy.csproj [hunk 4]
```diff
@@ -1,60 +1,56 @@
...
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+        <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+        <RepositoryUrl>https://github.com/Azure/azure-api-management-policy-toolkit</RepositoryUrl>
+        <Description>Azure API Management Policy Toolkit is a library allowing creation of policy documents in code.</Description>
+        <PackageTags>azure api-gateway api-management policy policies policy-toolkit</PackageTags>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <TargetFramework>.net8</TargetFramework>
```
Comment (tomkerkhove) [inline src/Pack/Policy/Policy.csproj:16]: Do we need .NET 8 or can we use lower? Customers most likely are not on 8 yet

File: src/Pack/Policy/Policy.csproj (hunk 3 above)
Comment (Mielek) [reply src/Pack/Policy/Policy.csproj:4]: I would go to with the Authoring suffix as this library is for Authoring policies. How do you like that?
```suggestion
        <PackageId>Azure.ApiManagement.PolicyToolkit.Authoring</PackageId>
```

Comment (tomkerkhove) [reply src/Pack/Policy/Policy.csproj:4]: Fine by me

Comment (tomkerkhove) [reply src/Pack/Policy/Policy.csproj:4]: Let's reflect this in namespaces as well though

Comment (Mielek) [reply src/Pack/Policy/Policy.csproj:4]: Ok, let's address this in next PR. It looks like packaging and packages needs to be revamped.

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**